### PR TITLE
Story/tyta 27/implement testing library

### PR DESCRIPTION
**Pull request by kilianpichard**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+  res.send('Hello World!');
+});
```
Comment (remirubis) [inline src/app.js:8]: Can we delete this part if it not used ?

File: src/server.js [hunk 2]
```diff
@@ -13,17 +10,17 @@ const DB = process.env.DB_URL;
...
+    teamRoute(app);
 
-// Start the server
-app.listen(PORT, () => {
-  console.log(`Server listening at http://localhost:${PORT}`);
-});
+    app.listen(PORT, () => {
+      console.log(`Server listening at http://localhost:${PORT}`);
+    });
+  })
```
Comment (remirubis) [inline src/server.js]: Finally we put it in the then ?

**Review by remirubis — COMMENTED** [review]
Just some questions

File: src/server.js (hunk 2 above)
Comment (kilianpichard) [reply src/server.js]: yes, I followed a tutorial for implement jest correctly with mongoose and they were doing like this (but we can rollback if needed)

File: src/app.js (hunk 1 above)
Comment (kilianpichard) [reply src/app.js:8]: hummm, we are using it for the first test, maybe we can remove it after ?

**Review by remirubis — DISMISSED** [review]
LGTM! 🚀

**Review by remirubis — APPROVED** [review]
LGTM! 🚀

**Pull request merged**